.github/workflows/codegen.yaml

```diff
@@ -0,0 +1,30 @@
+name: CI
+on: pull_request
+jobs:
+  codegen:
+    name: codegen check
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Cancel Previous Actions
+      uses: styfle/cancel-workflow-action@0.3.1
+      with:
+        workflow_id: 1442930
```

Comment:
this number should probably be different than the gloo one

Reply:
Good catch, thanks. Where does this come from?

Reply:
after reading https://github.com/styfle/cancel-workflow-action#advanced , i think we should just remove this line as we don't use it

Reply:
This works with styfle/cancel-workflow-action@0.4.1 as shown in those docs but not with 0.3.1 (I saw `##[error]Input required and not supplied: workflow_id`)